### ISSUE TITLE
Test API endpoints

### ISSUE THREAD

**Issue by RMeissnerCC** [openeduhub/metaqs-main#93]
A lot of bugs appear only when accessing the API direclty. Thus, testing those endpoints is valuable.

**Comment by MRuecklCC** [comment]
Yep, that definitely helps. We can some setup code from here :-)
https://github.com/openeduhub/metalookup/blob/main/tests/integration/metalookup_api_test.py

**Comment by MRuecklCC** [comment]
There now is also a consistency test that runs directly through the API: https://github.com/openeduhub/metaqs-main/blob/dev/tests/test_real_data.py.
It needs a running local service and is skipped in the CI.
